packages/frontend/src/components/common/ErrorState/index.tsx

```diff
@@ -56,7 +49,12 @@ const ErrorState: FC<{
         }
     }, [error]);
 
-    return <StyledSuboptimalState $hasMarginTop={hasMarginTop} {...props} />;
+    return (
+        <SuboptimalState
+            style={{ marginTop: hasMarginTop ? '20px' : undefined }}
+            {...props}
+        />
```

Comment:
underneath `SuboptimalState` is Mantine's Stack but it does not forward props correctly.

ideally SuboptimalState's prop type should be 

`interface Props extends StackProps { ... }`

and rest + sx spread


```
{...rest}
sx={{
  ...rest?.sx
}}
```

Reply:
Updated. I was trying not to change this thing cause it's used everywhere and its very hard to verify. But yolo. It's better to fix it, good call.